version.go

```diff
@@ -15,6 +16,10 @@ const CurrentVersionNumber = "0.23.0-dev"
 
 const ApiVersion = "/kubo/" + CurrentVersionNumber + "/" //nolint
 
+// EncodedAbsolutePathVersion is the version from which the absolute path header in
+// multipart requests is %-encoded. Before this version, its sent raw.
+var EncodedAbsolutePathVersion = semver.MustParse("0.23.0-dev")
```

Comment:
nit: having this here next to global version values feels confusing/error-prone (someone can do blind search/replace, or be tired and bump this while updating new release).

Could we move this to `core/commands/add.go` since this does not matter outside that RPC command anyway?